pkg/cloud/aws/commands/run_test.go

```diff
@@ -982,41 +982,6 @@ func Test_Run(t *testing.T) {
 		allServices  []string
 		inputData    string
 	}{
-		{
-			name: "fail without region",
-			options: flag.Options{
-				RegoOptions: flag.RegoOptions{SkipPolicyUpdate: true},
-			},
-			want:      "",
-			expectErr: true,
-		},
-		{
-			name: "fail without creds",
-			options: flag.Options{
-				RegoOptions: flag.RegoOptions{SkipPolicyUpdate: true},
-				AWSOptions: flag.AWSOptions{
-					Region: "us-east-1",
-				},
-			},
-			want:      "",
-			expectErr: true,
-		},
-		{
-			name: "try to call aws if cache is expired",
-			options: flag.Options{
-				RegoOptions: flag.RegoOptions{SkipPolicyUpdate: true},
-				AWSOptions: flag.AWSOptions{
-					Region:   "us-east-1",
-					Services: []string{"s3"},
-					Account:  "12345678",
-				},
-				CloudOptions: flag.CloudOptions{
-					MaxCacheAge: time.Minute,
-				},
-			},
-			cacheContent: "testdata/s3onlycache.json",
-			expectErr:    true,
-		},
 		{
```

Comment:
Did you miss moving this test case over?

Reply:
@simar7 
This test was successful because an error occurred due to the lack of credentials. I don't think this test is relevant at the moment, because if the cache has expired, then this [error is ignored](https://github.com/aquasecurity/trivy/blob/main/pkg/cloud/aws/cache/cache.go#L75) and all passed services are scanned.